package.json

```diff
@@ -53,7 +53,8 @@
     "prelint": "prettier --check --ignore-path .gitignore .",
     "postversion": "git push --follow-tags",
     "format": "eslint --ignore-path .gitignore --fix .",
-    "preformat": "prettier --write --ignore-path .gitignore ."
+    "preformat": "prettier --write --ignore-path .gitignore .",
+    "generate:readme": "tsx ./src/scripts/generate-readme.ts"
```

Comment:
Can you rename this and also move this above `format` (helps w/ future merge conflicts).

You can just update tsconfigs
![image](https://github.com/seamapi/javascript-next/assets/721372/7a36fc7f-87f1-4e61-a275-172f04648c10)


```suggestion
    "generate:readme": "tsx generate-readme.ts"
```

Reply:
done! also tsconfig updated auomatically 😮